testsuite/MDAnalysisTests/topology/test_top.py

```diff
@@ -348,6 +349,40 @@ class TestPRMNCRST(TOPBase):
     atom_i_improper_values = ()
 
 
+class TestPRMNCRST_negative(TOPBase):
```

Comment:
you should be able to write some sort of test that uses `assert_warns` (or is it `with pytest.warns:`?)